_minutes/2022-09-15-wecg.md

```diff
@@ -0,0 +1,135 @@
+# WECG Meetings 2022, Public Notes, Sep 15
+
+ * Chair: Simeon Vincent
+ * Scribes: Rob Wu
+
+Time: 8 AM PDT = https://everytimezone.com/?t=63226b00,384
+Call-in details: [WebExtensions CG, 15th September 2022](https://www.w3.org/events/meetings/d7bbce8f-549f-46ea-b440-ea6902f8707c/20220915T080000)
+Zoom issues? Ping `@robwu` (Rob Wu) in [chat](https://github.com/w3c/webextensions/blob/main/CONTRIBUTING.md#joining-chat)
+
+
+## Agenda: [github issues](https://github.com/w3c/webextensions/issues)
+
+The meeting will start at 3 minutes after the hour.
+
+ * **Carry-over from previous meetings**
+   * None
+ * **Other new issues**
+   * [Issue 258](https://github.com/w3c/webextensions/issues/258): Per-extension language preferences
+   * [Issue 269](https://github.com/w3c/webextensions/issues/269): declarativeNetRequest API: have isUrlFilterCaseSensitive condition set to false by default
+   * [Issue 270](https://github.com/w3c/webextensions/issues/270): Inconsistency: devtools.panels.create()
+   * [Issue 271](https://github.com/w3c/webextensions/issues/271): Proposal: extend the manifest file so that extensions can extend the permissions API
+   * [Issue 259](https://github.com/w3c/webextensions/issues/259) and [Issue 272](https://github.com/w3c/webextensions/issues/272): Proposal: browser.alert.create()
+ * **Check-in on ongoing issues**
+   * [Issue 162](https://github.com/w3c/webextensions/issues/162): DNR: disable individual static rules
+ * **Open discussion queue (add yourself at the bottom)**
+   * None
+
+
+## Attendees (sign yourself in)
+
+ 1. Rob Wu (Mozilla)
+ 2. David Johnson (Apple)
+ 3. Simeon Vincent (Chrome)
+ 4. Felipe Erias (Igalia)
+ 5. Jack Steam (Merlyn Mind)
+ 6. Matt Gibson (Bitwarden)
+ 7. Timothy Hatcher (Apple)
+ 8. Mukul Purohit (Microsoft)
+ 9. Tim Heflin (Keeper)
+ 10. James Hycner (Keeper)
+ 11. Oliver Dunk (1Password)
+ 12. Luca Greco (Mozilla)
+ 13. Joe Medley (Chrome)
+ 14. Carlos Jeurissen (Jeurissen Apps)
+
+
+## Meeting notes
+
+[there were zoom connectivity issues at the start of the meeting; meeting started ten minutes later than usual]
+
+[Issue 258](https://github.com/w3c/webextensions/issues/258): Per-extension language preferences
+
+ * [carlos] (summarizes issue and recent comment at https://github.com/w3c/webextensions/issues/258#issuecomment-1248028674)
```

Comment:
Summary of the issue is here https://github.com/w3c/webextensions/issues/258 , not that comment. I reorganized the proposal on 2022-09-12.

Reply:
During the meeting, I have explained your issue in detail. Explained the fact you drew inspiration from the handling on Android, how there are use cases to have an app / extension in a different language than the OS / browser. And the proposed methods / solutions proposed by you (@hanguokai).

Only later after comments from Rob and Simeon, I have explained some points made in my comment. This happened in between:
[timothy] It would also leak to the browser UI (button labels etc).
[simeon] What is “create instance” in i18n.createLanguageInstance referring to?

Between those comments, @xeenon was asking for the getMessage syntax he found in this issue before. On which I mentioned the fact the original post first mentioned the getMessage-extension and was later extended / modified by you (@hanguokai) to include the other proposals (setCurrentLanguage, getCurrentLanguage, and so on). After which I mentioned the alternative syntax as is now described in #274.

Reply:
@carlosjeurissen Sorry, I didn't attend this meeting. And thanks for the explanation.